requirements.txt

```diff
@@ -8,3 +8,5 @@ Pygments~=2.1
 PyPrint~=0.2.6
 requests~=2.12
 setuptools>=17.0
+attrs~=16.3.0
+
```

Comment:
that sounds really strange and this shouldn't happen. Please remove this line. If CI fails I'll look at it :+1: